62_CAD/common.hlsl

```diff
@@ -49,6 +50,19 @@ struct QuadraticBezierInfo
     float32_t phaseShift;
     float32_t stretchValue;
 };
+
+struct FontGlyphInfo
+{
+    float64_t2 topLeft; // 2 * 8 = 16 bytes
+    float32_t2 dirU; // 2 * 4 = 8 bytes (24)
+    float32_t aspectRatio; // 4 bytes (32)
+    // unorm8 minU;
+    // unorm8 minV;
+    // uint16 textureId;
+    uint32_t minUVTextureID; // 4 bytes (36)
```

Comment:
`minUV_textureID_packed` or some good convention for our packed values

Reply:
Done